instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/CamelRoutePolicy.java

```diff
@@ -23,11 +23,15 @@
 
 package io.opentelemetry.javaagent.instrumentation.apachecamel;
 
+import static io.opentelemetry.javaagent.instrumentation.apachecamel.MessageAttributeNamesFactory.withTextMapPropagatorFields;
+
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.api.trace.SpanBuilder;
 import io.opentelemetry.context.Context;
+import org.apache.camel.Endpoint;
 import org.apache.camel.Exchange;
 import org.apache.camel.Route;
+import org.apache.camel.component.aws.sqs.SqsEndpoint;
```

Comment:
Using a class from `camel-aws` in `camel-core` instrumentation will cause muzzle failures in applications that have core but not aws lib -- consider extracting this to a separate instrumentation; or using `MethodHandle`, but a separate instrumentation seems like a cleaner solution in this case.